platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -554,18 +570,22 @@ public class SwirldsPlatform implements Platform, PlatformWithDeprecatedMethods,
 
         reconnectThrottle = new ReconnectThrottle(settings.getReconnect());
 
+        final BasicConfig basicConfig = platformContext.getConfiguration().getConfigData(BasicConfig.class);
+
         topology = new StaticTopology(
                 selfId,
                 initialAddressBook.getSize(),
                 settings.getNumConnections(),
-                !settings.getChatter().isChatterUsed());
+                // unidirectional connections are ONLY used for old-style syncs, that don't run as a protocol
+                !settings.getChatter().isChatterUsed() && !basicConfig.syncAsProtocolEnabled());
 
         fallenBehindManager = new FallenBehindManagerImpl(
                 selfId,
                 topology.getConnectionGraph(),
                 this::checkPlatformStatus,
                 () -> {
-                    if (!settings.getChatter().isChatterUsed()) {
+                    // if we are using old-style syncs, early out and don't start the reconnect controller
```

Comment:
"early out"?

Reply:
I guess this expression isn't as common as I thought. I will change the verbiage